### DISCUSSION TITLE
Update from 3.0.8 to 3.0.9 results in component: [navigation-menu] not found

### DISCUSSION THREAD

**Discussion by KarstenSiebert** [livewire/livewire#7094] [7. Bugs (archived)]
### Livewire version

3.0.8 to 3.0.9 update

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

composer update

Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 3 updates, 0 removals
  - Upgrading league/mime-type-detection (1.13.0 => 1.14.0): Extracting archive
  - Upgrading livewire/livewire (v3.0.8 => v3.0.9): Extracting archive
  - Upgrading spatie/flare-client-php (1.4.2 => 1.4.3): Extracting archive
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   INFO  Discovering packages.  

  barryvdh/laravel-dompdf ..................................................................................................................... DONE
  jenssegers/agent ............................................................................................................................ DONE
  laravel/fortify ............................................................................................................................. DONE
  laravel/jetstream ........................................................................................................................... DONE
  laravel/sail ................................................................................................................................ DONE
  laravel/sanctum ............................................................................................................................. DONE
  laravel/tinker .............................................................................................................................. DONE
  livewire/livewire ........................................................................................................................... DONE
  nesbot/carbon ............................................................................................................................... DONE
  nunomaduro/collision ........................................................................................................................ DONE
  nunomaduro/termwind ......................................................................................................................... DONE
  pierre-adrien/laravel-cookie-consent ........................................................................................................ DONE
  simplesoftwareio/simple-qrcode .............................................................................................................. DONE
  spatie/laravel-ignition ..................................................................................................................... DONE
  spatie/laravel-permission ................................................................................................................... DONE

php artisan cache:clear

php artisan view:clear

 component:[navigation-menu] not found

![error-message](https://github.com/livewire/livewire/assets/113031423/85f1e0ed-28cb-4247-a98f-0d123af09fb0)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Ruben190404** [comment]
Roll back to `livewire v3.0.8`. The problem is already being [worked on](https://github.com/livewire/livewire/pull/7076) so it shouldn't take long before the fix is rolled out.

**Comment by KarstenSiebert** [comment]
Brilliant, thank you.

**Comment by KarstenSiebert** [comment]
Updated to 3.0.10, no issues.

**Comment by KarstenSiebert** [comment]
Closed
